### DISCUSSION TITLE
Regarding the Data for Building RAG

### DISCUSSION THREAD

**Discussion by siddartha-10** [microsoft/AI-Chat-App-Hack#7] [General]
I have quick question so the data set to is of our choice? or will there be any specific dataset's that will be provided to us during the HACK

**Comment by pamelafox** [comment]
We have a few example data sets (folders of PDFs) but we're also excited to see alternate data sets! You're welcome to use our example data sets if you want to hack on some other aspect of the RAG chat experience, but we're excited to see other data sets as well.

**Comment by siddartha-10** [comment]
When will those be uploaded? Thank you for responding.

**Reply by multispark** [reply]
We will be using the following app solution as a baseline to hack: https://github.com/Azure-Samples/azure-search-openai-demo/

The sample data sets are located in the data folder!
